CDCgov/NEDSS-Modernization
# Refarctored event search folder

## Description

Refactored Event Search Code base

## Tickets

* https://cdc-nbs.atlassian.net/browse/CNFT1-1772

## Steps to Verify
* Login to NBS
* Navigate to Event Search Page
* Everything should work as is after the code refactoring

## Checklist before requesting a review
